Sources/Tokenizers/Tokenizer.swift

```diff
@@ -161,6 +160,20 @@ public protocol Tokenizer {
     ) throws -> [Int]
 }
 
+extension Tokenizer {
+    public func applyChatTemplate(messages: [[String: String]]) throws -> [Int] {
+        try applyChatTemplate(messages: messages, chatTemplate: nil, addGenerationPrompt: true, truncation: false, maxLength: nil, tools: nil)
+    }
+
+    public func applyChatTemplate(messages: [[String: String]], chatTemplate: ChatTemplateArgument) throws -> [Int] {
+        try applyChatTemplate(messages: messages, chatTemplate: chatTemplate, addGenerationPrompt: true, truncation: false, maxLength: nil, tools: nil)
+    }
+
+    public func applyChatTemplate(messages: [[String: String]], chatTemplate: String) throws -> [Int] {
+        try applyChatTemplate(messages: messages, chatTemplate: .literal(chatTemplate), addGenerationPrompt: true, truncation: false, maxLength: nil, tools: nil)
+    }
+}
+
```

Comment:
See comment in [`TokenizersTemplates`](https://github.com/huggingface/swift-transformers/pull/168/files#diff-d09b4e8a4286c2839f012b3e4d37d16468e70380679bb78808e0033210091b80R18)